make-checkout

```diff
@@ -44,6 +44,8 @@ def main():
     if not opts.revision:
         opts.revision = "FETCH_HEAD"
 
+    base = os.environ.get("BASE_BRANCH")
```

Comment:
TBH I'd rather move away from environment variables in favor of options. It seems weird to mix the two. Let's please also make this a bit more descriptive, perhaps `--rebase BRANCH`?